Casks/f/fossa.rb

```diff
@@ -0,0 +1,18 @@
+cask "fossa" do
+  version "3.8.18"
+  sha256 "8fba90188752cea2c35a764f3b2308adc71058d7984cd761dd447a1c8359444f"
+
+  url "https://github.com/fossas/fossa-cli/releases/download/v#{version}/fossa_#{version}_darwin_amd64.zip",
+      verified: "github.com/fossas/fossa-cli/"
+  name "fossa"
+  desc "Zero-configuration polyglot dependency analysis tool"
+  homepage "https://fossa.com/"
+
+  binary "fossa"
+
+  caveats do
+    requires_rosetta
+  end
+
+  # No zap stanza required
```

Comment:
No zap should go before `caveats` (where it would go if there was a zap)

```suggestion
  binary "fossa"

  # No zap stanza required

   caveats do
     requires_rosetta
   end
```